spec/v3/services/env_var/delete_spec.rb

```diff
@@ -28,6 +28,6 @@
     end
 
     example { expect(last_response.status).to eq 200 }
-    example { pending 'should we return an empty body here?' }
+    example { expect(JSON.parse(last_response.body)["id"]).to eq(env_var[:id]) }
```

Comment:
We don't seem to document it and the question is somewhat broader, I guess. HTTP spec says that a DELETE request is allowed to return a body, but the client should not expect it. I can't think of a case where I'd want to delete something and then get info about the deleted thing back as a response. I'd be in favour of a 200 response with no body, I think. The fewer expectations we can give to the client when we're not really sure, the better. What do you reckon?